Bottom: be4804d68af5d78b9644864d04ef4d4797cf1663
Top:    fe0375d245c3eb6bfd28450fb7ad44554554988a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 01:33:30 +0100

Refresh of hid-gtk-implement-an-api-to-dr

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 8ace9d5..b8ca134 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -264,7 +264,12 @@ node_selection_changed_cb (GtkTreeSelection * selection, gpointer data)
   dup_string (&node_name, node->ListEntry);
   node_selected_net = selected_net;
 
+  /* Now just toggle a select of the node on the layout
+   */
   SelectPin (node, true);
+  IncrementUndoSerialNumber ();
+
+  /* And lead the user to the location */
   if (SeekPad (node, &conn, false))
     switch (conn.type) {
       case PIN_TYPE:
@@ -286,7 +291,6 @@ node_selection_changed_cb (GtkTreeSelection * selection, gpointer data)
           break;
         }
     }
-
 }
